workspaces/github-actions/plugins/github-actions/src/alpha/entityContent.test.tsx

```diff
@@ -50,13 +56,36 @@ describe('Entity content extension', () => {
           ).reactElement()}
         </EntityProvider>
       </TestApiProvider>,
+      {
+        mountedRoutes: {
+          '/test': convertLegacyRouteRef(rootRouteRef),
+        },
+      },
     );
 
-    await waitFor(
-      () => {
-        expect(screen.getByText('backstage/backstage')).toBeInTheDocument();
```

Comment:
Noting for readers that this test didn't behave as expected, it hid an issue where routing didn't work and the page blew up after initial render.